package.json

```diff
@@ -64,7 +64,7 @@
     "mixpanel-browser": "^2.46.0",
     "ngx-color": "7.0.0",
     "rudder-sdk-js": "^2.32.0",
-    "rxjs": "~6.6.7",
+    "rxjs": "^7.4.0",
```

Comment:
Why switch from `~` to `^`?

Reply:
^ -> `will update to all future minor/patch versions, without incrementing the major version` which is fine I think.